contracts/exchange/test/internal.ts

```diff
@@ -6,12 +6,14 @@ import {
     expect,
     hexRandom,
     LogDecoder,
+    OrderFactory,
```

Comment:
Can we do this without `OrderFactory` or any signature validation? This feels a lot less like unit tests if we need to validate signatures as well.

Reply:
I think that I can probably remove this. I was just using it for convenience, but I will remove

Reply:
Just override `_isValidOrderWithHashSignature()` in the contract to get around it.